packages/gatsby-theme-api-docs/src/components/api/resource/method/method.js

```diff
@@ -65,95 +71,91 @@ const Method = ({ apiKey, url, resourceUriParameters, method, methodType }) => {
-            <Parameters
-              title={pathParametersTitle}
-              parameters={allUriParameters}
-            />
-          ) : null}
-
-          {method.queryParameters ? (
-            <Parameters
-              title={queryParametersTitle}
-              parameters={method.queryParameters}
-            />
-          ) : null}
-
-          {method.body ? (
-            <RequestRepresentation
-              titleSuffix={requestRepresentation}
-              apiKey={apiKey}
-              apiType={method.body.applicationjson.type}
-            />
-          ) : null}
-        </SpacingsStack>
+    <SpacingsStack scale="s">
+      <SpacingsInline alignItems="center" scale="s">
+        <Type
+          css={css`
+            background-color: ${methodColor};
+          `}
+        >
+          {methodType}
+        </Type>
+
+        <TitleWithAnchor id={id}>{method.displayName}</TitleWithAnchor>
+      </SpacingsInline>
+
+      <Description>{method.description}</Description>
+
+      <SpacingsStack scale="m">
+        <UrlScopesResponseContainer
+          css={css`
+            border-left-color: ${methodColor};
+          `}
+        >
+          <UrlScopesResponses
+            data={{
+              apiKey,
+              uris,
+              scopes: {
+                title: oauth2Scopes,
+                scopes: method.securedBy
+                  ? method.securedBy[0].oauth_2_0.scopes
+                  : null,
+              },
+              responses: method.responses,
+            }}
+          />
+        </UrlScopesResponseContainer>
+
+        {allUriParameters.length > 0 ? (
+          <Parameters
+            title={pathParametersTitle}
+            parameters={allUriParameters}
+          />
+        ) : null}
+
+        {method.queryParameters ? (
+          <Parameters
+            title={queryParametersTitle}
+            parameters={method.queryParameters}
+          />
+        ) : null}
+
+        {method.body ? (
+          <RequestRepresentation
+            titleSuffix={requestRepresentation}
+            apiKey={apiKey}
+            apiType={method.body.applicationjson.type}
+          />
+        ) : null}
       </SpacingsStack>
-    </div>
+    </SpacingsStack>
   );
 };
 
-function computeMethodNameBackgroundColor(methodName) {
+function computeMethodColor(methodName) {
   switch (methodName) {
     case 'post':
-      return css`
-        background-color: ${colors.light.methods.post};
-      `;
+      return colors.light.methods.post;
     case 'delete':
-      return css`
-        background-color: ${colors.light.methods.delete};
-      `;
+      return colors.light.methods.delete;
     default:
-      return css`
-        background-color: ${colors.light.methods.get};
-      `;
+      return colors.light.methods.get;
   }
 }
 
 Method.propTypes = {
   apiKey: PropTypes.string.isRequired,
-  url: PropTypes.string.isRequired,
+  uris: PropTypes.object.isRequired,
```

Comment:
Please always try to define the shape of the objects (`PropTypes.shape`). It also helps eslint to catch errors